src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManager.groovy

```diff
@@ -71,6 +71,18 @@ class LSFJobManager extends AbstractLSFJobManager {
         super(executionService, parms)
     }
 
+    @Override
+    boolean checkCompatibilityToHost() {
```

Comment:
The name of this method does not express its function. "compatibility" can be anything (Windows? Linux? Right distribution? LSF Version? etc.).

Please change it to e.g. `submissionHostCanSubmit` or similar.